SUMMARY.md

```diff
@@ -148,6 +148,7 @@
   - [Template API Modules](libs/application/template-api-modules/README.md)
   - [Template Loader](libs/application/template-loader/README.md)
   - [Templates](libs/application/templates/README.md)
+    - [Application Templates Alcohol Tax Redemption](libs/application/templates/alcohol-tax-redemption/README.md)
```

Comment:
Please use update the readme header to be `Alcohol Tax Redemption` or use `<!-- gitbook-navigation: "Alcohol Tax Redemption" -->` to shorten the navigation label in gitbook.